ui/js/blocks/src/blocks/components/PodsServerSideRender.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * External dependencies
+ */
+import { debounce, isEqual } from 'lodash';
+import parse from 'html-react-parser';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { Placeholder, Spinner } from '@wordpress/components';
+import { InnerBlocks } from '@wordpress/editor';
+
+export function rendererPath(block, attributes = null, urlQueryArgs = {}) {
```

Comment:
Can you run eslint on this file? It looks like maybe some different formatting standards were applied to it.

Reply:
Done